lib/reporting/proofing_rate_report.rb

```diff
@@ -84,55 +86,42 @@ def to_csv
       end
     end
 
+    def sub_reports
+      @sub_reports ||= by_month ? by_month_ranges : trailing_days_ranges
+    end
+
     def reports
       @reports ||= begin
-        sub_reports = [0, *DATE_INTERVALS].each_cons(2).map do |slice_end, slice_start|
-          time_range = if slice_end.zero?
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day,
-                         )
-                       else
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day - 1.day,
-                         )
-                       end
-          Reporting::IdentityVerificationReport.new(
-            issuers: nil, # all issuers
-            time_range: time_range,
-            cloudwatch_client: cloudwatch_client,
-          )
-        end
+        if parallel?
```

Comment:
Nit: This path is never used except in a test. I think it's fine to leave in hopes of implementing, but just want to call that out.

Reply:
maybe we just remove it then forever since it's not used in prod

Reply:
I wasn't sure if you had intentions to some day implement it. I like the _idea_ of it. Barring any objection, I'll rip it out; it'll always remain in git history.

Reply:
I think we had to turn if off because of rate limiting issues, but yes it's dead code we can recover from git history